src/main/java/teammates/ui/controller/AdminInstructorAccountAddAction.java

```diff
@@ -297,37 +298,21 @@ private String getDemoCourseIdRoot(String instructorEmail){
     private String generateNextDemoCourseId(String instructorEmailOrProposedCourseId, int maximumIdLength){
         final boolean isFirstCourseId = instructorEmailOrProposedCourseId.contains("@");
         if(isFirstCourseId){
-            return trimCourseIdToMaximumLengthIfNecessary(getDemoCourseIdRoot(instructorEmailOrProposedCourseId)
-                    , maximumIdLength);
+            return StringHelper.truncateHead(getDemoCourseIdRoot(instructorEmailOrProposedCourseId)
+                                            , maximumIdLength);
```

Comment:
A comma is not considered as a operator. You can refer to our Java coding standard at 
`3. Use line wrapping to improve readability.` of https://docs.google.com/document/pub?id=1iAESIXM0zSxEa5OY7dFURam_SgLiSMhPQtU0drQagrs&embedded=true